### ISSUE TITLE
ModSpec: Conformance tests with more than one corresponding Requirement show no correspondence

### ISSUE THREAD

**Issue by manuelfuenmayor** [metanorma/metanorma-iso#808]
In relation to ISO 19115-3: https://github.com/metanorma/iso-19115-3/pull/18

Taking "https://standards.iso.org/iso19115/-1/1/req/metadata-xml/identification/associated-resource" as example, this requirement
shares target with another requirement:

```yaml
    - name: Validate with Schematron
      identifier: https://standards.iso.org/iso19115/-1/1/conf/metadata-xml/identification/schematron-rules
      targets:
      - https://standards.iso.org/iso19115/-1/1/req/metadata-xml/identification/topic-category
      - https://standards.iso.org/iso19115/-1/1/req/metadata-xml/identification/associated-resource
      method: Validate with mri.sch
```

In cases like this, a mesage error rises:

````
Requirements: (XML Line 002256):
Requirement https://standards.iso.org/iso19115/-1/1/req/metadata-xml/identification/associated-resource
has no corresponding Conformance test

````

**Comment by ronaldtse** [comment]
Document: https://github.com/metanorma/iso-19115-3/

**Comment by opoudjis** [comment]
````
targets:
      - https://standards.iso.org/iso19115/-1/1/req/metadata-xml/identification/topic-category
      - https://standards.iso.org/iso19115/-1/1/req/metadata-xml/identification/associated-resource
````

is not output Metanorma has been programmed to implement, and you cannot make up new markup, without speaking to me, and expect it to work:

* there is no "targets" supported for Modspec by Metanorma, only "target"
* there is no processing of lists, and the parallel of "inherits" is explicit that such values are repeated as key-value pairs in Metanorma, repeating the left-hand value in the definitions list.

Metanorma therefore only supports the following:

````
target:: https://standards.iso.org/iso19115/-1/1/req/metadata-xml/identification/topic-category
target:: https://standards.iso.org/iso19115/-1/1/req/metadata-xml/identification/associated-resource
````

I don't know where this YAML came from, but it has nothing to do with me, and I'm not inclined to support it. 

When I get around to this, I will implement multiple values being rendered delimited by `<br/>`, as I have already done for multiple provisions.

**Comment by ronaldtse** [comment]
Look at the liquid, the code is correct.

**Comment by ronaldtse** [comment]
Fixed in #813 . Thanks!
